TWLight/users/tests.py

```diff
@@ -337,6 +337,25 @@ def test_withdraw_application(self):
         app.refresh_from_db()
         self.assertEqual(app.status, Application.INVALID)
 
+    def test_withdraw_application_date_closed(self):
```

Comment:
Since the difference between this test and the one above is only the `date_closed` assertion, I suggest you add that assertion in the previous test, with a comment that this assertion is included after T269730 bug fix.

Reply:
ok @suecarmol, I will edit that and will update you soon. :)

Reply:
Hi I have made changes according to your suggestion, thanks :)